Sources/XcbeautifyLib/String+Colored.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension String {
+    func coloredTime() -> String {
+        guard let time = Double(self) else { return self }
+        if time < 0.025 { return self }
+        if time < 0.100 { return f.Yellow }
+        return f.Red
+    }
+
+    func coloredDeviation() -> String {
+        guard let deviation = Double(self) else { return self }
+        if deviation < 1 { return self }
+        if deviation < 10 { return f.Yellow }
+        return f.Red
+    }
```

Comment:
Would these functions be clearer if written as switch statements and (open) ranges?

Reply:
This is the existing implementation from `TerminalRenderer`, and - in this specific case - I don't think it'd be any clearer with just a few branches.